apps/prairielearn/src/question-servers/freeform.ts

```diff
@@ -1126,27 +1127,27 @@ export async function render(
 
       const extensions = context.course_element_extensions;
       const dependencies = {
-        coreStyles: [],
-        coreScripts: [],
-        nodeModulesStyles: [],
-        nodeModulesScripts: [],
-        coreElementStyles: [],
-        coreElementScripts: [],
-        courseElementStyles: [],
-        courseElementScripts: [],
-        extensionStyles: [],
-        extensionScripts: [],
-        clientFilesCourseStyles: [],
-        clientFilesCourseScripts: [],
-        clientFilesQuestionStyles: [],
-        clientFilesQuestionScripts: [],
+        coreStyles: [] as string[],
+        coreScripts: [] as string[],
+        nodeModulesStyles: [] as string[],
+        nodeModulesScripts: [] as string[],
+        coreElementStyles: [] as string[],
+        coreElementScripts: [] as string[],
+        courseElementStyles: [] as string[],
+        courseElementScripts: [] as string[],
+        extensionStyles: [] as string[],
+        extensionScripts: [] as string[],
+        clientFilesCourseStyles: [] as string[],
+        clientFilesCourseScripts: [] as string[],
+        clientFilesQuestionStyles: [] as string[],
+        clientFilesQuestionScripts: [] as string[],
       };
       const dynamicDependencies = {
         nodeModulesScripts: {},
         coreElementScripts: {},
         courseElementScripts: {},
         extensionScripts: {},
-        clientFilesCourseScripts: {},
+        clientFilesCourseScripts: {} as Record<string, string>,
```

Comment:
👀 I'm confused why only this one had to change?

Reply:
This was flagged by https://typescript-eslint.io/rules/restrict-template-expressions

Reply:
In any case, I fixed that in ed32c0fd05e656102949ea224dc4336fbd3fba0f. We can look at those type issues separately, as that is a deeper issue.